app/src/features/surveys/view/survey-animals/AnimalSection.tsx

```diff
@@ -389,7 +389,7 @@ export const AnimalSection = (props: IAnimalSectionProps) => {
           {props.critter.captures.map((capture) => (
             <Collapse key={capture.capture_id}>
               <EditDeleteStubCard
-                header={formatDate(new Date(capture.capture_timestamp))}
+                header={formatDate(new Date(capture.capture_date))}
```

Comment:
I think this needs to first combine the captureDate and captureTime

Reply:
I think this is deprecated.

Reply:
ah right